cumulus_library/studies/core/builder_condition.py

```diff
@@ -54,6 +54,12 @@ def denormalize_codes(self):
                 "http://snomed.info/sct",
                 "http://hl7.org/fhir/sid/icd-10-cm",
                 "http://hl7.org/fhir/sid/icd-9-cm",
+                "http://hl7.org/fhir/sid/icd-9-cm/diagnosis",
+                # Cerner specific systems
+                "http://terminology.hl7.org/CodeSystem/data-absent-reason",
+                # EPIC specific systems
```

Comment:
I'm guessing we can't say anything else about them? But it'd be nice if we could distinguish what these two mean even a little bit.

Reply:
yeah, hard agree!